core/src/trezor/ui/layouts/mercury/__init__.py

```diff
@@ -139,6 +139,20 @@ def confirm_multisig_warning() -> Awaitable[None]:
     )
 
 
+if not utils.BITCOIN_ONLY:
```

Comment:
Code organization wise, is there a reason you don't put `confirm_unknown_contract_warning` and `confirm_other_data` under already existing `if not BITCOIN_ONLY` branch. Now it's 3x in this file. Similar situation in `tt/__init__.py` and `tr/__init__.py`

Reply:
I also debated with myself where it is better - under the same `if` or under a separate one, but close to the functions that are similar. I think it can be argued either way. I can move if you prefer the other way.

Reply:
Should be fine either way.